test/nix/x86_64-prebuilt-pkgs.nix

```diff
@@ -0,0 +1,35 @@
+{ pkgs ? import <nixpkgs> { }, }:
+with pkgs;
+let
+  inherit busybox;
+  fio = callPackage ./fio.nix { };
+  hackbench = callPackage ./hackbench.nix { };
+  inherit iperf3;
+  lmbench = callPackage ./lmbench.nix { };
+  ltp = callPackage ./ltp.nix { };
+  inherit memcached;
+  inherit nginx;
+  inherit redis;
```

Comment:
Some packages such as redis appear both in the Nix file, an input of the `nix-build` command, and in the Docker file, as the input of the `nix-env --install` command. Does this mean that the `nix-env --install` command for these packages is NOT necessary?